autonomy/data/Dockerfiles/agent/start.sh

```diff
@@ -21,9 +21,11 @@ else
     then
         echo "Generating the fresh key with a password!"
         aea generate-key ethereum --password $AEA_PASSWORD
+        aea generate-key ethereum-flashbots --password $AEA_PASSWORD
```

Comment:
Isn't this going to end up being a different key from the one above? I think what we need to do is generate once, and copy with a different name for the flashbots (if a different file name is required).

Reply:
I would also prefer that.

cc @angrybayblade

Reply:
just copy the `ethereum_private_key.txt` to `ethereum_flashbots_private_key.txt`

Reply:
Addressed on 6e4f9c2e29e7826151a36013b400f6ed3eb62bc5.